gcp/table_gcp_kubernetes_cluster.go

```diff
@@ -0,0 +1,424 @@
+package gcp
+
+import (
+	"context"
+	"strings"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+	"google.golang.org/api/container/v1"
+)
+
+func tableGcpKubernetesCluster(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "gcp_kubernetes_cluster",
+		Description: "GCP Kubernetes Cluster",
+		List: &plugin.ListConfig{
+			Hydrate: listKubernetesClusters,
+		},
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"name", "location"}),
+			Hydrate:    getKubernetesCluster,
+		},
+		Columns: []*plugin.Column{
+			// commonly used columns
+			{
+				Name:        "name",
+				Description: "The name of this cluster.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "self_link",
+				Description: "Server-defined URL for the resource.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "location_type",
+				Description: "Location type of the cluster",
```

Comment:
```suggestion
				Description: "Location type of the cluster.",
```

Reply:
@rajeshbal65 Can you also mention the valid values (i.e. `Regional/Zonal`) in column description?